docker/charts/templates/controller-deployment.yaml

```diff
@@ -12,11 +12,13 @@ spec:
     metadata:
       labels: {{ include "chart.controller.labels" . | nindent 8}}
     spec:
+      {{- if .Values.devMode.createConsoleHostPathPV.enabled }}
       volumes:
-        - name: controller-storage
+        - name: console-hostpath-volume
           hostPath:
-            path: {{ .Values.storage.agentHostPathRoot }}/controller
+            path: {{ .Values.devMode.createConsoleHostPathPV.rootPath }}/controller/console
```

Comment:
Can we remove the word 'PV' from the field name as it seems to be only a 'host path' and not a 'PV'?

Reply:
hostPath is a typed volume of PersistentVolume ?

Reply:
updated